src/spawn.cpp

```diff
@@ -170,7 +170,7 @@ bool Spawns::loadFromXml(const std::string& filename, bool isCalledByLua)
 					continue;
 				}
 
-				Npc* npc = Npc::createNpc(nameAttribute.as_string());
+				auto npc = Npc::createNpc(nameAttribute.as_string());
```

Comment:
Could this be `const auto`
```suggestion
				const auto npc = Npc::createNpc(nameAttribute.as_string());
```